src/components/pages/PowerheadPage/view.js

```diff
@@ -40,20 +42,21 @@ function renderPeopleStatsList(report, monthIndex) {
   const extFteVal = Math.ceil(report.ext_fte[monthIndex]);
   return (
     <ul className={styles.peopleStats}>
-      {renderPeopleStatsItem('Total exts.', extFteVal, 'FTE', true)}
-      {renderPeopleStatsItem('Total ints.', totalIntsVal, 'FTE')}
-      {renderPeopleStatsItem('Booked', bookedVal, 'FTE')}
-      {renderPeopleStatsItem('Bench', benchVal, 'FTE')}
+      {renderPeopleStatsItem('Total exts.', extFteVal, 'FTE', 'exts', true)}
+      {renderPeopleStatsItem('Total ints.', totalIntsVal, 'FTE', 'ints')}
+      {renderPeopleStatsItem('Booked', bookedVal, 'FTE', 'booked')}
+      {renderPeopleStatsItem('Bench', benchVal, 'FTE', 'bench')}
     </ul>
   );
 }
 
-function renderFinancialsStatsItem(label, value, legendBarStyle, special = false) {
```

Comment:
5 arguments starts to smell bad, but I'm not really sure what to do.
At least we should try to combine legendBarStyle and special into just one `style` argument which can be set on the `<li>`, not on the div for the legend. So then in SCSS we can set both the legend style and the span styles by using nested SCSS rules.